### DISCUSSION TITLE
CORS-related error prevents me adding a subdomain to my micro

### DISCUSSION THREAD

**Discussion by bhavesh-k** [deta/cloud-docs#485] [Help]
<img width="1437" alt="image" src="https://user-images.githubusercontent.com/7852169/190561427-462054a1-df03-4fde-9415-f65c8d7f1a40.png">

See the screenshot for the full error as logged in the browser console when I clicked the "Save" button trying to add a subdomain to my project.

Looks like a new issue, please help :)

**Comment by jnsougata** [comment]
![image](https://user-images.githubusercontent.com/53375272/190561813-db80b800-5ed0-4445-a11c-aba281d564e1.png)
